src/DIRAC/WorkloadManagementSystem/Executor/JobScheduling.py

```diff
@@ -60,7 +60,7 @@ def optimizeJob(self, jid, jobState):
             return S_ERROR("RescheduleCounter has to be an integer")
         if reschedules != 0:
             delays = self.ex_getOption("RescheduleDelays", [60, 180, 300, 600])
-            delay = delays[min(reschedules, len(delays) - 1)]
+            delay = delays[min(reschedules - 1, len(delays) - 1)]
             waited = toEpoch() - toEpoch(fromString(attDict["RescheduleTime"]))
```

Comment:
```suggestion
            waited = toEpoch() - toEpoch(attDict["RescheduleTime"])
```
As in the PR description, not sure if this will always work, assuming there was a reason for `fromString` to be there.

Reply:
Hard to say what's correct without seeing the code in action. This line was there also in rel-v7r3 though.

Reply:
Ok, then let's leave it as is with the fromString. Or people see issues in v7r3 as well?

Reply:
In v7r2 I see from `dirac-wms-job-attribues` `'RescheduleTime': '2022-09-29 01:26:25',`
Maybe a difference between mysql and elastic backend of jobattributes? (or was that just job parameters)? Or mysql version? or python package version? or actual code change somewhere else in DIRAC.

Reply:
Elastic is only for JobParameters.

Reply:
dirac-wms-job-attributes for 8.1.0a1 for lbcertifdirac gives
```
 'RescheduleTime': datetime.datetime(2022, 9, 29, 9, 52, 34),
```

Reply:
Can you profit of this PR to make `fromString` smart enough to recognize that what is passed in is already in the right format?

Reply:
I did already do that, or what do you mean?

Reply:
Oh sorry you are right, then everything's OK.